python-black.el

```diff
@@ -119,6 +119,22 @@ DISPLAY-ERRORS is non-nil, shows a buffer if the formatting fails."
   (and (= (point-min) beg)
        (= (point-max) end)))
 
+(defun python-black-block ()
```

Comment:
this is a very generic name that may confuse people. `python-black-org-mode-block` perhaps?

Reply:
I'll rename it.